lib/gauge.js

```diff
@@ -95,11 +102,7 @@ class Gauge extends Metric {
 		};
 	}
 
-	async get() {
-		if (this.collect) {
-			const v = this.collect();
-			if (v instanceof Promise) await v;
-		}
+	_getSync() {
```

Comment:
ditto private comment (won't repeat in the others - but applies to all new `_*` methods)